great_expectations/experimental/datasources/postgres_datasource.py

```diff
@@ -36,16 +36,18 @@ class BatchRequestError(Exception):
 class ColumnSplitter:
     column_name: str
     method_name: str
-    param_names: List[str]
+    param_names: Sequence[str]
```

Comment:
```
postgres_datasource.py:48: error: Incompatible types in assignment (expression has type "List[Literal['year', 'month']]", base class "ColumnSplitter" defined the type as "List[str]")  [assignment]
postgres_datasource.py:48: note: "List" is invariant -- see https://mypy.readthedocs.io/en/stable/common_issues.html#variance
postgres_datasource.py:48: note: Consider using "Sequence" instead, which is covariant
```

Reply:
This was originally a Sequence but I ran into some issues. Looks like they are resolved now if this works. That's good.

Reply:
pydantic will serialize it as a list.
https://docs.pydantic.dev/usage/types/#typing-iterables

But I think mypy will complain if you mutate a sequence.